configure.ac

```diff
@@ -11,8 +11,8 @@ AC_PREREQ([2.63])
 define([scm_r], esyscmd([sh -c "git rev-parse --short=7 HEAD"]))
```

Comment:
maybe we should package into tarball VERSION file w/ git hash used during creation of tarball, and if git not found - use it.